src/assets/specs/README.md

```diff
@@ -0,0 +1,42 @@
+## Generators
+
+### To generate go code:
+
+oapi-codegen src/assets/specs/command-api.yaml > api/commands.go
+
+#### Dependencies:
+
+* opai-codegen: To install:
```

Comment:
We can use `go run …` directly; see also how we do it for the moby-openapi stuff.

Reply:
How?

```
$ go run github.com/deepmap/oapi-codegen/cmd/oapi-codegen -h
no required module provides package github.com/deepmap/oapi-codegen/cmd/oapi-codegen: go.mod file not found in current directory or any parent directory; see 'go help modules'
```

Reply:
Hmm, right, we don't actually have it pinned anywhere (that is, no `go.mod` that depends on it, since we don't have _any_ uses of it at this point).

But then, it also makes no sense to generate go code without any way to _use_ that go code…

I guess we'll need to install it manually for now, then; can we at least pin it to a version? And then once we actually consume it with some go code, bake that into a `go:generate` with `go run…` (because at that point we'd have a `go.mod`).

Reply:
I'm not generating the go code in this PR -- just leaving a note for myself or someone else suggesting a way to do it in the future.